packages/wonder-blocks-link/src/components/link-core.tsx

```diff
@@ -240,6 +252,7 @@ const _generateStyles = (
             // TODO(WB-1521): Update the underline offset to be 4px after
             // the Link audit.
             // textUnderlineOffset: 4,
+            textUnderlineOffset: 1.5,
```

Comment:
**question:** any particular reason why you choose 1.5 over 1? same question goes for 3.5 vs 3 in the icon style.

Reply:
I was messing with decimals to get a closer match, but noticed they cut off part of the parenthesis when used inline, so I just changed them to whole numbers again.